clang/lib/APINotes/APINotesWriter.cpp

```diff
@@ -1146,6 +1146,12 @@ class TagTableInfo : public CommonTypeTableInfo<TagTableInfo, TagInfo> {
 
     writer.write<uint8_t>(Flags);
 
+    if (auto Copyable = TI.isSwiftCopyable()) {
```

Comment:
Unnecessary braces and we should be able to use the named constants here.

Reply:
Fixed